meson_options.txt

```diff
@@ -24,7 +24,7 @@ option('man',
 
 option('tests',
   type: 'boolean',
-  value: true,
+  value: false,
```

Comment:
Breaking due to a missing dependency on `gir1.2-eosmetrics`? Is this still needed since you added that in the -debian branch?

Reply:
No, it's broken *because* of this dependency. One of the CSS unit tests ends up trying to open a connection to the metrics daemon, which emits a warning and fails the tests.